clipping_manager/managers.py

```diff
@@ -27,4 +27,15 @@ def random(self, limit=None):
 class BookQuerySetManager(models.QuerySet):
 
     def for_user(self, user):
-        return self.filter(user=user)
\ No newline at end of file
+        return self.filter(user=user)
+
+class MyClippingsFilesManager(models.Manager):
+    def create_file(self, content, language_header):
+        # Make sure file creation proceeds
+        # even if language_header > field's max_length
+        if language_header:
+            language_header = language_header if len(language_header) < 255 else None
```

Comment:
Why not just trim it to 255 characters (`language_header[:255]`)? Should still have more value than None, right?

Reply:
xd idk, you are absolutely right